webapp/src/Entity/Submission.php

```diff
@@ -89,6 +89,13 @@ class Submission extends BaseApiEntity implements ExternalRelationshipEntityInte
     #[Serializer\Expose(if: "context.getAttribute('domjudge_service').checkrole('jury')")]
     private ?string $entry_point = null;
 
+    #[ORM\Column(
+        nullable: false,
+        options: ['comment' => 'Whether this submission was imported during shadowing but had an error while doing so.']
+    )]
+    #[Serializer\Groups([ARC::GROUP_NONSTRICT])]
+    private bool $importError = false;
```

Comment:
Can we make this a string, not a boolean? Or add an extra string? That contains the actual error?